kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -50,9 +49,12 @@ def create_connector(
         """
         config_json = kafka_connect_config.dict(exclude_none=True)
         connect_data = {"name": connector_name, "config": config_json}
-        response = httpx.post(
+
+        client = httpx.AsyncClient()
```

Comment:
It would make more sense to add the host in the constructor:
```python
httpx.AsyncClient(base_url=host)
```

Reply:
Or is it necessary to close the client every time you do the call?

Reply:
the mock with the context manager makes things more unclear when testing, that's why I choose to use the api without context manager.

Reply:
I tried it out, and the tests worked for me. This other mock library uses `base_url`.
https://github.com/lundberg/respx#pytest--httpx

Reply:
I wouldn't use the context manager in this case and rather do what @raminqaf initially recommended

Reply:
you mean the client inside self ? @disrupted